pkg/internal/testhelpers/containers.go

```diff
@@ -22,7 +22,13 @@ import (
 
 const (
 	defaultDB       = "postgres"
-	connectTemplate = "postgres://postgres:password@%s:%d/%s"
+	connectTemplate = "postgres://%s:password@%s:%d/%s"
+
+	postgresUser = "postgres"
+	promUser     = "prom"
+
+	SUPERUSER   = true
```

Comment:
I think it is idiomatic for this to be `Superuser` and `NonSuperuser` all caps not Idiomatic